NEWS.md

```diff
@@ -1,3 +1,28 @@
+# simtrial 0.4.0
+
+## New features
+
+- RMST test is available (#188, thanks @LittleBeannie).
+- Milestone test is available (#199, #204, #211, #237, thanks @LittleBeannie).
+- Test version of `sim_gs_n()` is available with coming unit testing (#195, #201, #208, #212, thanks @LittleBeannie and @jdblischak).
+- Users have the flexibility to create custom interim and final analyses cuttings according to their specific requirements by utilizing the `create_cutting_test()` function. (#215, #221, thanks @jdblischak).
+- Users can create various testing approaches for interim and final analyses by utilizing the `create_test()` function. (#221, thanks @jdblischak).
+- Users have the option to perform multiple tests on a simulated dataset using the `multitest()` function (#215, thanks @jdblischak).
```

Comment:
I suggest somehow noting that `multitest()` is still somewhat experimental. I created it prior to your standardization of the test functions in #227, so we may want to improve it in the future

Reply:
Great suggestion, I almost added your original words as a note after this item.